### PR TITLE
Fix/wdgets/hook replacement

### PR DESCRIPTION
**Pull request by SeverS**
## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

**Comment by SeverS** [comment]
to be merged after #1971

File: ui/widgets/sidebar/src/components/sidebar-new.tsx [hunk 1]
```diff
@@ -1,202 +0,0 @@
...
-        </Box>
-        <Box customStyle="w-fit flex flex-grow flex-col">
-          <Text variant="button-md">{title}</Text>
-          <Text variant="footnotes1" customStyle="text-grey5">
-            {subtitle}
-          </Text>
-        </Box>
-        <Box customStyle="w-fit h-fit ml-6 self-center">
-          {loginQuery.data?.id && (
-            <Button icon="PowerIcon" size="xs" iconOnly={true} onClick={handleLogout} />
```
Comment (didd) [inline ui/widgets/sidebar/src/components/sidebar-new.tsx]: Sever , I see this file is conflicting on this PR with the one on next. You can go ahead with your changes. On my side I tried to add copy to clipboard feature and truncate to the DID. However, we want to replace part of the DID key with an icon and a proper truncate based on the type of the key so my change will not be relevant.

Comment (SeverS) [reply ui/widgets/sidebar/src/components/sidebar-new.tsx]: I will include the changes

**Review by kenshyx — DISMISSED** [review]

File: ui/widgets/top-bar/src/components/index.tsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import ReactDOM from 'react-dom/client';
```
Comment (kenshyx) [inline ui/widgets/top-bar/src/components/index.tsx]: hmm, this won't be captured by webpack for externals extraction

Comment (SeverS) [reply ui/widgets/top-bar/src/components/index.tsx]: good catch, I was just testing...

**Review by kenshyx — APPROVED** [review]

**Pull request merged**